fhevm/tfhe_wrappers.c

```diff
@@ -0,0 +1,2124 @@
+#include "tfhe_wrappers.h"
```

Comment:
Not related to this PR, but with so much C code, we need to add some tooling to check for memory errors/leaks.

Reply:
For the first point, I can look into that, and also make sure the Go build does use hardening flags during compilation maybe.

For the second point, I need to look deeper into the wrappers to see if there is anything we can automate here.